CHANGELOG.md

```diff
@@ -155,6 +155,7 @@ New deprecation(s):
 
 ### Other
 
+- **General**: Added RBAC permissions for list & watch LimitRange, and check default limits from LimitRange for validations ([#5377](https://github.com/kedacore/keda/pull/5377))
```

Comment:
Nit, would be better to move this to Improvements section, as it is user facing change :)

Reply:
Makes sense. Added. Thanks!

And with that, I realized, :sweat_smile: I should probably add a note in the docs about the controller picking the default limits from `LimitRange` for validation.